graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/GraphQLDomain.java

```diff
@@ -15,30 +15,18 @@
  */
 package com.expediagroup.streamplatform.streamregistry.graphql.model;
 
+import java.util.List;
+
 import lombok.Builder;
 import lombok.Value;
 
-import com.expediagroup.streamplatform.streamregistry.model.Domain;
-
 @Value
 @Builder
 public class GraphQLDomain {
   String name;
   String owner;
   String description;
-  Iterable<GraphQLKeyValue> tags;
+  List<GraphQLKeyValue> tags;
```

Comment:
Are we sure about this? Seems like tags should be a unique set.
Why the change from Iterable  to List, does BUL require this?

Reply:
Yes, BUL requires the change in type. But yes you're right - it should be a Set with comparator on the key.

Reply:
We can look at that separately from this PR